types/liferay/liferay.d.ts

```diff
@@ -0,0 +1,100 @@
+declare const Liferay: {
```

Comment:
Could that be moved from this file to `index.d.ts`?

Reply:
I removed the export and moved the definition to the index.d.ts. 👍